src/main/java/com/bieganski/ox/Field.java

```diff
@@ -12,11 +12,15 @@
         this.symbol = symbol;
     }
 
-
     boolean isOnPosition(int position) {
         return this.position == position;
     }
 
+    boolean isInBounds(int bounds) {
+        return position < bounds && position >= 0;
```

Comment:
I would change order to 0 <= position && position < bounds (it resembles standard notation of:
0 <= p < b